st2auth/requirements.txt

```diff
@@ -7,6 +7,7 @@
 # update the component requirements.txt
 bcrypt==3.2.0
 eventlet==0.30.2
+git+https://github.com/pimguilherme/st2-auth-backend-sso-saml2.git@feat/saml#egg=st2-auth-backend-sso-saml2
```

Comment:
we must rememberd to change this to the default repository after merging the other repo!